x-pack/test/apm_api_integration/tests/transactions/latency.spec.ts

```diff
@@ -77,73 +68,92 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
   registry.when(
     'Latency with a basic license when data is loaded',
-    { config: 'basic', archives: [archiveName] },
+    { config: 'basic', archives: [] },
     () => {
+      const GO_PROD_RATE = 80;
+      const GO_DEV_RATE = 20;
+      const GO_PROD_DURATION = 1000;
+      const GO_DEV_DURATION = 500;
+      before(async () => {
+        const serviceGoProdInstance = apm
+          .service({ name: serviceName, environment: 'production', agentName: 'go' })
+          .instance('instance-a');
+        const serviceGoDevInstance = apm
+          .service({ name: serviceName, environment: 'development', agentName: 'go' })
+          .instance('instance-b');
+
+        await synthtraceEsClient.index([
+          timerange(start, end)
+            .interval('1m')
+            .rate(GO_PROD_RATE)
```

Comment:
Short-hand (and it has the added benefit of ingesting more frequently to avoid spikes every minute)
```suggestion
            .ratePerMinute(GO_PROD_RATE)
```